<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo Articulo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Manual</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Autor</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Journal</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>AÃ±o</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Titulo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Tab1_Autor">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Tab1_Journal">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Tab1_Year">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Tab1_Titulo">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>180</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn_Crear1">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>210</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Crear Articulo</string>
       </property>
      </widget>
      <widget class="QLabel" name="l_autor">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="l_journal">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>80</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="l_year">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>130</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="l_titulo">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>180</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Descarga</string>
      </attribute>
      <widget class="QPushButton" name="btn_seleccionar">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>161</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>Seleccionar Archivo .bib</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_archivos">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>60</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn_Crear2">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>210</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Crear Articulo</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Source</string>
      </attribute>
      <widget class="QPushButton" name="Btn_Crear3">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>210</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Crear Articulo</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="te_source">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>561</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
